include/hpp/bezier-com-traj/solve_end_effector.hh

```diff
@@ -309,66 +315,75 @@ void computeJerkCostFunctionDiscretized(int numPoints,const ProblemData& pData,d
 
 
 
-
-
-template <typename Path>
-ResultDataCOMTraj solveEndEffector(const ProblemData& pData,const Path& path, const double T, const double weightDistance, bool useVelCost){
-
-
-    if(verbose)
-      std::cout<<"solve end effector, T = "<<T<<std::endl;
-    assert (weightDistance>=0. && weightDistance<=1. && "WeightDistance must be between 0 and 1");
-    double weightSmooth = 1. - weightDistance;
-    std::vector<bezier_t::point_t> pi = computeConstantWaypoints(pData,T);
+std::pair<MatrixXX,VectorX> computeEndEffectorConstraints(const ProblemData& pData, const double T,std::vector<bezier_t::point_t> pi){
     std::vector<waypoint_t> wps_jerk=computeJerkWaypoints(pData,T,pi);
     std::vector<waypoint_t> wps_acc=computeAccelerationWaypoints(pData,T,pi);
     std::vector<waypoint_t> wps_vel=computeVelocityWaypoints(pData,T,pi);
     // stack the constraint for each waypoint :
-    MatrixXX A;
-    VectorX b;
-    Vector3 jerk_bounds(10000,10000,10000);
+    Vector3 jerk_bounds(10000,10000,10000);    //TODO : read it from somewhere (ProblemData ?)
     Vector3 acc_bounds(10000,10000,10000);
     Vector3 vel_bounds(10000,10000,10000);
-    const int DIM_VAR = dimVar(pData);
+    MatrixXX A;
+    VectorX b;
     computeConstraintsMatrix(pData,wps_acc,wps_vel,acc_bounds,vel_bounds,A,b,wps_jerk,jerk_bounds);
-  //  std::cout<<"End eff A = "<<std::endl<<A<<std::endl;
- //   std::cout<<"End eff b = "<<std::endl<<b<<std::endl;
-    // compute cost function (discrete integral under the curve defined by 'path')
-    MatrixXX H_rrt,H;
-    VectorX g_rrt,g;
-    std::pair<MatrixXX,VectorX> Hg_smooth;
+    return std::make_pair<MatrixXX,VectorX>(A,b);
```

Comment:
Can you use `std::make_pair(A, b);` here ? The templated version doesn't work in c++11.

Reply:
Done, thanks for noticing this.